### DISCUSSION TITLE
Orange pi series

### DISCUSSION THREAD

**Discussion by Tozzigh** [Joshua-Riek/ubuntu-rockchip#147] [Q&A] [answered]
I want to run emulators till ps2, considering also the new 5 plus 16gb what is the best Orange Pi that I can buy? Is it better to take the 5 32gb?

**Comment by Joshua-Riek** [comment, marked as answer]
All the Orange Pi 5 devices have the same CPU and GPU performance. The only real difference is the Orange Pi 5 Plus has PCIE 3.0 for faster SSD speeds and more IO. I would not go with the 5 Plus for your use case. Also, choose an 8GB or 16GB model. There is no 32GB model because the memory chips are way too expensive, 32GB is advertised for marketing purposes.
